docs/docsite/rst/scenario_guides/guide_scaleway.rst

```diff
@@ -0,0 +1,231 @@
+- ``commercial_type`` represents the name of the commercial offers.
+  You can check out the Scaleway pricing page to find which instance is right for you.
+
+Take a look at this short playbook to see a working example using ``scaleway_compute``:
+
+.. code-block:: yaml
+
+    # SCW_TOKEN='XXX' ansible-playbook ./test/legacy/scaleway_compute.yml
+
+    - name: Test compute instance lifecycle on a Scaleway account
+      hosts: localhost
+      gather_facts: no
+      environment:
+        SCW_API_KEY: ""
+
+      tasks:
+
+        - name: Create a server
+          register: server_creation_task
+          scaleway_compute:
+            name: foobar
+            state: present
+            image: ec8b431e-ad39-4523-8b94-f3fa7f3cbd06
+            organization: 951df375-e094-4d26-97c1-ba548eeb9c42
+            region: ams1
+            commercial_type: START1-S
+            wait: true
+
+        - debug: var=server_creation_task
+
+        - assert:
+            that:
+              - server_creation_task is success
+              - server_creation_task is changed
+
+        - name: Run it
+          scaleway_compute:
+            name: foobar
+            state: running
+            image: ec8b431e-ad39-4523-8b94-f3fa7f3cbd06
+            organization: 951df375-e094-4d26-97c1-ba548eeb9c42
+            region: ams1
+            commercial_type: START1-S
+            wait: true
+            tags:
+              - web_server
+          register: server_run_task
+
+        - debug: var=server_run_task
+
+        - assert:
+            that:
+              - server_run_task is success
+              - server_run_task is changed
+
+
+Dynamic Inventory Script
+========================
+
+Ansible ships with `Scaleway Dynamic Inventory Plugin <https://docs.ansible.com/ansible/devel/plugins/inventory/scaleway.html>_`.
+You can now get a complete inventory of your Scaleway resources through this plugin and filter it on
+different parameters (``regions`` and ``tags`` are currently supported).
+
+Let's create an example!
+Suppose that we want to get all hosts that got the tag web_server.
+Create a file named ``scaleway_inventory.yml`` with the following content:
+
+.. code-block:: yaml
+
+    plugin: scaleway
+    regions:
+      - ams1
+      - par1
+    tags:
+      - web_server
+
+This inventory means that we want all hosts that got the tag ``web_server`` on the zone ``ams1`` and ``par1``.
+Once you have configured this file, you can get the information using the following command:
+
+::
+
+    $ ansible-inventory --list -i scaleway_inventory.yml
+
+    {
+        "_meta": {
+            "hostvars": {
+                "dd8e3ae9-0c7c-459e-bc7b-aba8bfa1bb8d": {
+                    "ansible_verbosity": 6,
+                    "arch": "x86_64",
+                    "commercial_type": "START1-S",
+                    "hostname": "foobar",
+                    "ipv4": "51.15.200.26",
```

Comment:
Change the IP address so it's not pointing to something real. You can safely use anything in the 192.0.2.0/24 range.  Similar comment for the organization UUID below if that is pointing to a real org.